build.sbt

```diff
@@ -266,8 +266,11 @@ lazy val akkaHttp = project
       "com.typesafe.akka"             %% "akka-http"                  % "10.2.4",
       "com.typesafe.akka"             %% "akka-serialization-jackson" % akkaVersion,
       "com.typesafe.akka"             %% "akka-stream"                % akkaVersion,
-      "de.heikoseeberger"             %% "akka-http-circe"            % "1.36.0" % Optional,
-      "de.heikoseeberger"             %% "akka-http-play-json"        % "1.36.0" % Optional,
+      "de.heikoseeberger"             %% "akka-http-circe"            % "1.36.0"   % Optional,
+      "de.heikoseeberger"             %% "akka-http-play-json"        % "1.36.0"   % Optional,
+      "de.heikoseeberger"             %% "akka-http-zio-json"         % "1.36.0"   % Optional,
+      "dev.zio"                       %% "zio-test"                   % zioVersion % "test",
```

Comment:
The CI is currently calling `akkaHttp/compile`, can you change it to `akkaHttp/test`? (2 occurences in .circleci/config.yml)

Reply:
I still don't see the test run, I think it needs `testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework"))` too

Reply:
You're right. Should be fixed now.